modules/dataframe/src/processors/dataframeexporter.cpp

```diff
@@ -148,7 +148,7 @@ void DataFrameExporter::exportAsCSV(bool separateVectorTypesIntoColumns) {
         const auto components = col->getBuffer()->getDataFormat()->getComponents();
         if (components > 1 && separateVectorTypesIntoColumns) {
             for (size_t k = 0; k < components; k++) {
-                oj = citation + col->getHeader() + ' ' + componentNames[k] + citation;
+                oj = fmt::format("{0}{} {}{0}", citation, col->getHeader(), componentNames[k]);
```

Comment:
Might add units too

Reply:
not that we parse then though....

Reply:
That's the thing. I would expect that a DataFrame which was exported to a file and then read again results in the same DataFrame, units in the header would change that. Unless we have support for parsing, as pointed out.
I'd move that to a future PR.